components/layout.tsx

```diff
@@ -1,18 +1,19 @@
 import React from "react";
 
 // Components
-import CardGrid from "./card-grid/card-grid";
 import Header from "./header/header";
 import Meta from "./meta";
 
-const Layout: React.FC = () => {
+const Layout: React.FC = (props) => {
   return (
     <>
       <Meta />
       <Header />
-      <CardGrid />
+      <div style={{marginTop: '4rem'}}>
```

Comment:
Doing it this way allows to wrap the children with the top margin so all content will be properly offset from the type

Reply:
great idea